Justfile

```diff
@@ -66,7 +81,7 @@ test-e2e: pnpm_install
     pnpm exec cypress run -r list -q
 
 # Run tier2 application regression tests
-test-e2e-tier2: pnpm_install
+test-e2e-tier2: pnpm_install deploy-mina-indexer && shutdown-mina-indexer
```

Comment:
suggestion(non-blocking): `pnpm_install deploy-mina-indexer && shutdown-mina-indexer` becomes an alias or command you can reference (since you do it in 2 places).

Reply:
same